<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_XIANGSHAN_setting</class>
 <widget class="QDialog" name="qt_XIANGSHAN_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>新建列</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>界面</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>计价秤</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>龙支付</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#ff546b;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QPushButton" name="pushButton_select_adv_path">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>添加广告图片</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>皮肤</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_fuping_skin">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>0</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_show_pic_list">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>120</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>广告图片列表</string>
           </property>
          </widget>
          <widget class="QTableView" name="tableView_pic_list">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>30</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_select_shop_info">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>添加店铺名片</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_upan">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>40</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_upan_refresh">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>40</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/image/view-refresh.png</normaloff>:/image/view-refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>选择u盘</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidget_upan_list">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>80</y>
             <width>430</width>
             <height>341</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">
QListWidget::item:selected{
        background: #ff546b;
}

QListWidget::item:!selected{
        background: transparent;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_pic_list_clearAll">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>310</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>全不选</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_pic_list_selectAll">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>270</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_shop_info_size">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>220</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>info size</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>230</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>230</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_scale_port">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>10</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_scale_bundrate">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>50</y>
             <width>180</width>
             <height>30</height>
            </rect>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>400</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>商户代码(MERCHANTID)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>400</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>分行代码(BRANCHID)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>400</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>柜台号(POSID)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>400</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>公钥(PUB)</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ccb_MERCHANTID">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>40</y>
             <width>600</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ccb_BRANCHID">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>100</y>
             <width>600</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ccb_POSID">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>160</y>
             <width>600</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ccb_PUB">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>220</y>
             <width>600</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ccb_QUPWD">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>280</y>
             <width>600</width>
             <height>30</height>
            </rect>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>250</y>
             <width>400</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>密码(QUPWD)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_ccb_default">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>320</y>
             <width>150</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>演示用例</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label_loading">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>loading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_msg">
       <property name="styleSheet">
        <string notr="true">color:#ff546b;</string>
       </property>
       <property name="text">
        <string>message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
